app/graphql/types/query_type.rb

```diff
@@ -2,23 +2,16 @@
 
 module Types
   class QueryType < Types::BaseObject
-    field :node, Types::NodeType, null: true, description: "Fetches an object given its ID." do
-      argument :id, ID, required: true, description: "ID of the object."
-    end
+    field :world_index, Types::WorldIndexType, null: false, description: "World index"
 
-    def node(id:)
-      context.schema.object_from_id(id, context)
+    def world_index
+      live_edition(content_id: "369729ba-7776-4123-96be-2e3e98e153e1")
     end
```

Comment:
Is there a way this could be made more generic, so we don't need to hardcode content IDs for each page GraphQL will serve?

I'm thinking maybe we could have the base path as a query field, then we can grab the live edition based on the base path instead?

Reply:
>Or just a generic edition query taking the base path?

That is more what I was thinking.  Maybe we could determine which type to use based on something we already know about that base path (e.g. the document type or schema name)?

Reply:
I've drafted a generic version here: https://github.com/alphagov/publishing-api/pull/2930

It's a little hacky under the hood, but the generic interface is probably nicer. Curious to hear your thoughts

I'll probably write some more unit tests if we do go down this avenue

Reply:
I much prefer the approach in #2930, as it'll allow us to roll out more content types without needing to define additional query parameters (unless actually needed for filtering).